forge.config.ts

```
@@ -9,10 +10,34 @@ import { FuseV1Options, FuseVersion } from "@electron/fuses";
 
 import { execSync } from "child_process";
 import path from "node:path";
-const gitCommit = execSync("git rev-parse --short HEAD").toString().trim();
+const gitBranch = execSync("git rev-parse --abbrev-ref HEAD").toString().trim();
+
 const now = new Date();
 const buildDate = now.toISOString().slice(0, 10).replace(/-/g, "");
 
+const packageJSON = JSON.parse(fs.readFileSync('./package.json', 'utf8'));
```

ESLint

[error] Replace `'./package.json',·'utf8'` with `"./package.json",·"utf8"`

(prettier/prettier)